lib/resourcemerge/imagestream.go

```diff
@@ -30,6 +30,7 @@ func EnsureImagestreamv1(modified *bool, existing *imagev1.ImageStream, required
 }
 
 func ensureTagReferencev1(modified *bool, existing *imagev1.TagReference, required imagev1.TagReference) {
+	ensureTagReferencev1Defaults(&required)
```

Comment:
Can I extort you into writing a unit test for `EnsureImagestreamv1` that would cover the basic cases (and the newly added one)? No unit tests so far are not your fault, but we can use the opportunity of changing the code to pay the debt.

Reply:
You got it :muscle:

Reply:
Tests added :wrench: